jkube-kit/build/api/src/main/java/org/eclipse/jkube/kit/build/api/auth/RegistryAuthConfig.java

```diff
@@ -13,103 +13,102 @@
  */
 package org.eclipse.jkube.kit.build.api.auth;
 
-import lombok.Builder;
```

Comment:
This class was completely reformatted and is really hard to track the changes (at least in the GH interface).
Could you please revert the reformatting so that it's clearer for us to see what was changed here?